contracts/standard/arbitration/CriptoKitties/Auction/ClockAuction.sol

```diff
@@ -0,0 +1,149 @@
+pragma solidity ^0.4.18;
```

Comment:
Could you indicate in header comments that it's cryptokitty code and not Kleros code?
Thanks.

Reply:
I've added comments in the beginning of each file mentioning that.